packages/guideline-blocks-settings/src/index.ts

```diff
@@ -75,34 +75,34 @@ export type BlockSettingsStructureExport = {
 export type BlockProps = {
     /**
      * The Frontify App Bridge provides an interface to the Frontify app internals.
-     * {@link https://developer.frontify.com/document/1366#/details-concepts/content-blocks/introducing-the-appbridge}
+     * {@link https://developer.frontify.com/d/XFPCrGNrXQQM/content-blocks#/details-concepts-1/content-blocks/introducing-the-app-bridge}
```

Comment:
Could you check to fix the slug `detail-concepts-1` to be cleaner like `detail-concepts` ? 🤔

Reply:
This might change later anyway.
According to @floriangaechter:
> We're planning on moving the developer platform to a dedicated tool anyway sometime this year and that would involve a new path-scheme (especially in regards to the coming additions to the brand sdk)